variables.tf

```diff
@@ -72,3 +72,14 @@ variable "threat_intel_list_path" {
   default     = ""
   type        = "string"
 }
+
+
+variable "accept_invite" {
+  description = "Accept invitation from a master account."
+  default     = false
+}
```

Comment:
Would you mind adding `type = "string"`? Not having it tickles my OCD.

Reply:
I can do. It was tickling mine too, but the default is string so didn't bother!